src/components/common/Stepper.vue

```diff
@@ -40,6 +40,7 @@ export default class Stepper extends Vue {
   readonly step1Valid!: boolean;
 
   // FUTURE - map these to states as above
+  @State(state => state.stateModel.addPeopleAndRoleStep.valid)
```

Comment:
Please also fix the Step 5 Valid logic (which is currently not working).

Reply:
implemented!